cdap-security/src/test/java/co/cask/cdap/security/authorization/AuthorizationCacheInvalidationTest.java

```diff
@@ -1,146 +0,0 @@
-/*
- * Copyright © 2016 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package co.cask.cdap.security.authorization;
-
-import co.cask.cdap.common.conf.Constants;
-import co.cask.cdap.common.test.AppJarHelper;
-import co.cask.cdap.proto.id.NamespaceId;
-import co.cask.cdap.proto.security.Action;
-import co.cask.cdap.proto.security.Principal;
-import co.cask.cdap.proto.security.Role;
-import co.cask.cdap.security.spi.authorization.Authorizer;
-import co.cask.cdap.security.spi.authorization.PrivilegesManager;
-import org.apache.twill.filesystem.Location;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
-
-import java.io.IOException;
-import java.util.Collections;
-import java.util.jar.Attributes;
-import java.util.jar.Manifest;
-
-/**
- * Tests invalidation of caches in {@link DefaultAuthorizationEnforcementService} and
- * {@link PrivilegesFetcherProxyService} when privileges are updated.
- */
-public class AuthorizationCacheInvalidationTest extends AuthorizationTestBase {
```

Comment:
removing this cache test makes sense but do we have test for the new cache we added ? I didnt see any unit test for them those caches.